eng/Version.Details.xml

```diff
@@ -5,6 +5,16 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>839cdfb0ecca5e0be3dbccd926e7651ef50fdf10</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23159.2">
```

Comment:
Have darc subscriptions been created for this and the new roslyn dependency?

Reply:
> Have darc subscriptions been created for this and the new roslyn dependency?

No, as I've noticed in some other per-repo prebuilt work, that subscriptions were created after PRs were merged. Perhaps it's irrelevant as it won't kick-in until these dependencies show up.

Do we also need coherency attributes for these new dependencies?

Reply:
I'm going to create a subscription for SBRP - should it be a daily or weekly subscription? I presume we don't want it to be manual.